src/custom/pages/Swap/index.tsx

```diff
@@ -86,6 +87,14 @@ const SwapModWrapper = styled(SwapMod)`
     .expertMode ${AutoRow} {
       padding: 0 1rem;
     }
+
+    ${Circle} {
+      color: ${({ theme, confirmed, disabled }) => (disabled ? theme.text1 : confirmed ? theme.white : theme.text1)};
```

Comment:
For some reason, it's not picking up on the `confirmed` condition and apply `theme.white` to be used in the green circle......